plugins/snmp_receiver_plugin/mibs/private/MELLANOX-DCB-TRAPS.mib

```diff
@@ -0,0 +1,235 @@
+MELLANOX-DCB-TRAPS-MIB DEFINITIONS ::= BEGIN 
```

Comment:
not sure we need to have this DCB traps mib as it is related to ETH related traps (not related to IB) - need to confirm that with Meir R.